doc/async_operations.md

```diff
@@ -1,5 +1,25 @@
 # Asynchronous operations
 
-You are reading a draft of the next documentation and it's in continuos update so
-if you don't find what you need please refer to:
-[C++ wrapper classes for the ABI-stable C APIs for Node.js](https://nodejs.github.io/node-addon-api/)
\ No newline at end of file
+Node.js native add-ons often need to execute long running tasks and to avoid of
+blocking the **event loop** they have to accomplish to them in the asynchronous way.
```

Comment:
have to accomplish to them -> have to run them asynchronously from the **event loop**.